fabtests/unit/cq_test.c

```diff
@@ -79,13 +79,17 @@ static int cq_open_close_simultaneous(void)
 		return -FI_ENOMEM;
 
 	ret = 0;
-	for (opened = 0; opened < count && !ret; opened++) {
+	while (opened < count) {
 		ret = create_cq(&cq_array[opened], 0, 0, FI_CQ_FORMAT_UNSPEC,
 				FI_WAIT_UNSPEC);
-		if (ret) {
+		if (ret)
 			ret = create_cq(&cq_array[opened], 0, 0,
 					FI_CQ_FORMAT_UNSPEC, FI_WAIT_NONE);
-		}
+
+		if (ret)
+			break;
```

Comment:
Adding these two lines to the original loop should achieve the same result. Can remove the redundant ` && !ret` condition, though.

Reply:
Done

Reply:
Updated